gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemBase.java

```diff
@@ -611,6 +612,24 @@ protected int getDefaultPort() {
     return result;
   }
 
+  public boolean hasPathCapability(Path path, String capability) throws IOException {
+    switch (validatePathCapabilityArgs(path, capability)) {
+        // TODO: remove string literals in favor of Constants in CommonPathCapabilities.java
```

Comment:
No, we won't be able to include that change. It would work for Hadoop 3, but Hadoop 2 doesn't have the `CommonPathCapabilities` class that defines the constants. (Notice that the patch was committed only to master, where we dropped Hadoop 2 support.)